packages/web-components/fast-foundation/src/slider/slider.template.ts

```diff
@@ -13,6 +13,7 @@ export const SliderTemplate = html<Slider>`
         class="${x => (x.readOnly ? "readonly" : "")} 
         ${x => x.orientation || Orientation.horizontal}"
         tabindex="${x => (x.disabled ? null : 0)}"
+        aria-valuetext="${x => x.valueText}"
```

Comment:
Was there a reason that this couldn't be implemented without the code below like this?

```js
aria-valuetext="${x => x.valueTextFormatter(x.value)}"
```

With this code, the `area-valuetext` attr would update any time the formatter or the value changed. You would just need to set the formatter to a default function that returns null, like so:

```ts
@observable valueTextFormatter: (value: string) => string | null = () => null;
```

Reply:
Updated to go with Rob's suggestions. Thanks!